charts/v2.0/cray-hms-capmc/Chart.yaml

```diff
@@ -13,6 +13,6 @@ dependencies:
 maintainers:
   - name: Hardware Management
     url: https://github.com/orgs/Cray-HPE/teams/hardware-management
-appVersion: "1.33.0"
+appVersion: "1.33.2"
```

Comment:
This violates our chart versioning rules. Also the CSM manifest defines a 1.33.1 application version override: https://github.com/Cray-HPE/csm/blob/release/1.2/manifests/sysmgmt.yaml#L51

If this fix is targeted for CSM 1.2, it will also have no affect as an override is already in place. Need to update the manifest with a different application version.